Rules/Rules.csproj

```diff
@@ -45,9 +45,10 @@
     <Compile Include="Rule\EnemyHealthScaledRule.cs" />
     <Compile Include="Rule\EnemyRespawnDisabledRule.cs" />
     <Compile Include="Rule\GoldPickedUpMultipliedRule.cs" />
+    <Compile Include="Rule\PieceConfigAdjustedRule.cs" />
     <Compile Include="Rule\RatNestsSpawnGoldRule.cs" />
     <Compile Include="Rule\SampleRule.cs" />
-    <Compile Include="Rule\PieceConfigAdjustedRule.cs" />
+    <Compile Include="Rule\SorcererStartCardsModifiedRule.cs" />
     <Compile Include="Rule\StartHealthAdjustedRule.cs" />
     <Compile Include="Rule\ZapStartingInventoryAdjustedRule.cs" />
```

Comment:
FYI @jimconner : I'm okay leaving this file indented this way.  My IDE isn't forcing changes, and that way we don't have to worry about fighting VS.